library/Fission/Config/Types.hs

```diff
@@ -3,16 +3,21 @@ module Fission.Config.Types (Config (..)) where
 
 import qualified Network.HTTP.Client as HTTP
 
+import Data.Pool
+import Database.Persist.Sql (SqlBackend)
+
+import qualified Network.AWS.Auth   as AWS
+import qualified Network.IPFS.Types as IPFS
+
 import           Fission.Prelude
 import           Fission.Web.Types
-import qualified Network.IPFS.Types            as IPFS
-import qualified Fission.Storage.Types         as DB
-import qualified Fission.Platform.Heroku.Types as Heroku
-import qualified Network.AWS.Auth              as AWS
-import qualified Fission.AWS.Types             as AWS
+import qualified Fission.Platform.Heroku.ID.Types       as Heroku
+import qualified Fission.Platform.Heroku.Password.Types as Heroku
+import qualified Fission.AWS.Types                      as AWS
+
 
 -- | The top level 'Fission' application 'RIO' configuration
-data Config dbBackend = Config
+data Config = Config
```

Comment:
No longer passing in a configurable backend, because the Persistent's `SqlBackend` will work with any SQL, not just PostgreSQL.

Reply:
aw yiss